### PR TITLE
Fix release steps/rules

### LINKED ISSUE #365: Fix release steps/rules
**Describe the bug**
Release rules are not working properly

**Environment information:**

- OS: Any
- Platform: Any
- Firmware Version: a079d88fd7a1b2ee518502007a879ea7f63bbbd1

**Steps to Reproduce**

1. Go to [release instructions](https://github.com/fibercrypto/skywallet-mcu/tree/75435fee9f6dc965c8a062844652f4436e0922c9#update-the-version).

**Actual behavior**
Instructions are not working.

**Expected behavior**
This should work.

PD: Fix `Run make build` too in readme.

### PR DESCRIPTION
**Pull request by stdevAlDen**
Fixes #365 

Changes:	
- Fix release instructions

Does this change need to mentioned in CHANGELOG.md?	
no	

Requires testing	
no

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -141,13 +141,13 @@ release-combined: release-bootloader release-firmware ## Build bootloader and fi
 	cp releases/skywallet-bootloader-no-memory-protect-v$(VERSION_BOOTLOADER).bin tiny-firmware/bootloader/combine/bl.bin
 	cp releases/skywallet-firmware-v$(VERSION_FIRMWARE).bin tiny-firmware/bootloader/combine/fw.bin
 	cd tiny-firmware/bootloader/combine/ ; $(PYTHON) prepare.py && \
-	mv tiny-firmware/bootloader/combine/combined.bin releases/skywallet-full-no-mem-protect-$(COMBINED_VERSION).bin
+	cp combined.bin ../../../releases/skywallet-full-mem-protect-$(COMBINED_VERSION).bin
```
Comment (olemis) [inline Makefile]: Parameterize `../../../releases` in global `RELEASE_PATH` variable.

**Review by olemis — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -79,15 +79,29 @@ before_deploy:
...
-    local_dir: releases
-    upload-dir: skywallet
-    acl: private
-    region: $AWS_REGION
-    on:
-      repo: skycoin/hardware-wallet
-      tags: true
+- provider: releases
+  api_key:
+    secure: TODO generate it using travis CLI
```
Comment (stdevAlDen) [inline .travis.yml:84]: @olemis fill this pls

File: .travis.yml [hunk 3]
```diff
@@ -79,15 +79,29 @@ before_deploy:
...
+  - releases/emulator-$(uname -s)-${TRAVIS_TAG}
+  - releases/version.txt
+  skip_cleanup: true
+  draft: true
+  on:
+    repo: fibercrypto/skywallet-mcu
+    tags: true
+- provider: releases
+  api_key:
+    secure: TODO generate it using travis CLI
```
Comment (stdevAlDen) [inline .travis.yml:99]: @olemis fill this pls

File: README.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 # Skycoin hardware wallet
 
-[![Build Status](https://travis-ci.com/skycoin/hardware-wallet.svg?branch=develop)](https://travis-ci.com/skycoin/hardware-wallet)
+[![Build Status](https://travis-ci.com/fibercrypto/skywallet-mcu.svg?branch=develop)](https://travis-ci.com/fibercrypto/skywallet-mcu)
 
 **Note**: This repo is no longer maintained. Please refer to the new [Skywallet](https://github.com/SkycoinProject/hardware-wallet) repo.
```
Comment (olemis) [inline README.md]: @stdevAlDen remove this line .

**Review by olemis — CHANGES_REQUESTED** [review]

**Review by olemis — APPROVED** [review]

**Pull request merged**